declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc

```diff
@@ -0,0 +1,72 @@
+
+[NOTE]
+====
+As a developer, if you are creating an Argo CD application and deploying cluster-wide resources, ensure that your cluster administrator grants the necessary permissions to them.
+
+Otherwise, after the Argo CD reconciliation, you will see an authentication error message in the application's `Status` field similar to the following example:
+
+.Example authentication error message
+
+[source,terminal]
+----
+persistentvolumes is forbidden: User "system:serviceaccount:gitops-demo:argocd-argocd-application-controller" cannot create resource "persistentvolumes" in API group "" at the cluster scope.
+----
+====
+
+[id="prerequisites_customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances"]
+== Prerequisites
+
+* You have installed {gitops-title} 1.13.0  or a later version on your {OCP} cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI.
+* You have installed a xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance] in your defined namespace `spring-petclinic`. 
+* You have validated that the user-defined cluster-scoped instance is configured with the cluster roles and cluster role bindings for the following components:
++
+** Argo CD Application Controller
+** Argo CD server
+** Argo CD ApplicationSet Controller (provided the ApplicationSet Controller is created)
++
+* You have deployed a `cluster-configs` xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#creating-an-application-by-using-the-argo-cd-dashboard_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[Argo CD application] with the `customclusterrole` link:https://github.com/redhat-developer/openshift-gitops-getting-started/tree/main/customclusterrole[path] in the `spring-petclinic` namespace and created the `test-gitops-ns` namespace and `test-gitops-pv` persistent volume resources.
++
+[NOTE]
+====
+The `cluster-configs` Argo CD application must be managed by a user-defined cluster-scoped instance with the following parameters set:
+
+* The `selfHeal` field value set to `true`
+* The `syncPolicy` field value set to `automated`
+* The  *Label* field set to the `app.kubernetes.io/part-of=argocd` value
+* The  *Label* field set to the `argocd.argoproj.io/managed-by=<user-defined_namespace>` value so that the Argo CD instance in your defined namespace can manage your namespace
```

Comment:
It can be difficult to parse replaceables when mixing hyphens and underscores. Consider just using underscores.
```suggestion
* The  *Label* field set to the `argocd.argoproj.io/managed-by=<user_defined_namespace>` value so that the Argo CD instance in your defined namespace can manage your namespace
```